apps/learner/src/lib/components/LearningUnit.svelte

```diff
@@ -68,7 +68,9 @@
         <span class="font-medium text-slate-950">Play</span>
       </button>
 
-      <div class="flex items-center gap-x-2">
+      <div class="ml-auto flex justify-between gap-x-2">
```

Comment:
Do we still need `ml-auto`?

Reply:
I changed the parent corner itself to use `items-center` and `justify-between` to center the play button and progress display vertically and horizontally on each edge (left and right)